examples/Remarketing/UploadStoreSalesTransactions.cs

```diff
@@ -1,490 +1,501 @@
+                        offlineUserDataJobType, externalId, advertiserUploadDateTime,
+                        bridgeMapVersionId, partnerId, customKey);
+
+                // Adds transactions to the job.                
+                AddTransactionsToOfflineUserDataJob(offlineUserDataJobServiceClient, customerId,
+                    offlineUserDataJobResourceName, conversionActionId);
+
+                // Issues an asynchronous request to run the offline user data job.
+                offlineUserDataJobServiceClient.RunOfflineUserDataJobAsync(
+                    offlineUserDataJobResourceName);
+
+                Console.WriteLine("Sent request to asynchronously run offline user data job " +
+                                  $"{offlineUserDataJobResourceName}.");
+
+                // Offline user data jobs may take up to 24 hours to complete, so instead of waiting
+                // for the job to complete, retrieves and displays the job status once and then
+                // prints the query to use to check the job again later.
+                CheckJobStatus(client, customerId, offlineUserDataJobResourceName);
+            }
+            catch (GoogleAdsException e)
+            {
+                Console.WriteLine("Failure:");
+                Console.WriteLine($"Message: {e.Message}");
+                Console.WriteLine($"Failure: {e.Failure}");
+                Console.WriteLine($"Request ID: {e.RequestId}");
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Creates an offline user data job for uploading store sales transactions.
+        /// </summary>
+        /// <param name="offlineUserDataJobServiceClient">The offline user data job service
+        ///     client.</param>
+        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
+        /// <param name="offlineUserDataJobType">The type of user data in the job (first or third
+        ///     party). If you have an official store sales partnership with Google, use
+        ///     StoreSalesUploadThirdParty. Otherwise, use StoreSalesUploadFirstParty or
+        ///     omit this parameter.</param>
+        /// <param name="externalId">Optional (but recommended) external ID to identify the offline
+        ///     user data job.</param>
+        /// <param name="advertiserUploadDateTime">Date and time the advertiser uploaded data to the
+        ///     partner. Only required if uploading third party data.</param>
+        /// <param name="bridgeMapVersionId">Version of partner IDs to be used for uploads. Only
+        ///     required if uploading third party data.</param>
+        /// <param name="partnerId">ID of the third party partner. Only required if uploading third
+        ///     party data.</param>
+        /// <param name="customKey">Optional: If uploading data with custom key and values, also specify this value.</param>

```

Comment:
Rephrase for consistency with other descriptions, perhaps "Custom key name. Only required if uploading data with custom key and values."